src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
+spring.datasource.username=${DB_USERNAME}
+spring.datasource.password=${DB_PASSWORD}
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
+
+server.port = ${PORT}
```

Comment:
add eof

Reply:
:thinking: eof == a empty line??

Reply:
claro si significa "end of file", es una buena práctica

Reply:
:+1: